### DISCUSSION TITLE
How to make a dynamic pattern with static options?

### DISCUSSION THREAD

**Discussion by Lanayx** [nuxt/nuxt#16772] [Questions]
I'm trying to match the following routes
http://localhost:3000/en
http://localhost:3000/fr
to one page. I started with the following structure 
pages ->
..[lang] ->
....index.vue
However I don't want anything to match this route, just en and fr (so http://localhost:3000/abcd should return 404), how do I achieve this?

**Comment by sevillaarvin** [comment]
How about implementing a middleware and abort the connection if not on the list of languages?
